templates/web/base/report/_item.html

```diff
@@ -2,7 +2,7 @@
 [% PROCESS 'admin/report_blocks.html' ~%]
 [% END ~%]
 
-[% IF c.user.has_permission_to('planned_reports', problem.bodies_str_ids) ~%]
+[% IF c.user.has_permission_to('planned_reports', problem.bodies_str_ids) OR c.user.is_planned_report(problem) ~%]
```

Comment:
There is also the remove button on a report's page (in `report/_main.html`).

Reply:
I think that's ok as it is?

Reply:
Oh yes, ignore me.